README.md

```diff
@@ -88,7 +88,7 @@ For more information, see [Linux & BSD installation](./docs/install_linux.md).
 | ------------------- | --------------------|
 | `winget install --id GitHub.cli` | `winget upgrade --id GitHub.cli` |
 
-> **Note**  
+> **Note**
```

Comment:
I think your editor was removing whitespace, which is totally legit, but I realize this is the older version of a markdown alert that was deprecated and should be switched to the supported format:

```suggestion
> [!NOTE]
```

Reply:
Done